Naming/src/main/java/com/epam/engx/cleancode/naming/task4/service/Notebook.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.engx.cleancode.naming.task4.service;
+
+
+import com.epam.engx.cleancode.naming.task4.thirdpartyjar.CustomerContact;
+
+public interface Notebook {
+    CustomerContact findCustomerContactByCustomerId(Long customerId);
```

Comment:
```suggestion
    CustomerContact findById(Long customerId);
```

Reply:
I don't wholly agree. Because:
1. The CustomerContact class has fields:
private Long customerId;
private String emailId;
The program is able to give incorrect result, if suddenly customerId of one Object matches with other object's emailId. So, in my opinion, it would be better not to use the overloaded method.
2. The class has been renamed Notebook. It possible includes as customers as another contact categories (suppliers, lawyers, staff and etc.). In case of a large number of objects,  it may be advisable to search for them by categories.